data/avro-schema/src/main/resources/avro/net/corda/data/flow/event/external/ExternalEventResponseExceptionEnvelope.avsc

```diff
@@ -0,0 +1,27 @@
+{
+  "type": "record",
+  "name": "ExternalEventResponseExceptionEnvelope",
```

Comment:
I don't think we should call this exception envelope, rather it's an external event error.

Reply:
Sure, will change.